src/Microsoft.TestPlatform.Build/Microsoft.TestPlatform.targets

```
@@ -70,37 +122,36 @@ Copyright (c) .NET Foundation. All rights reserved.
     <MSBuild Projects ="$(MSBuildProjectFullPath)" />
     <Microsoft.TestPlatform.Build.Tasks.VSTestLogsTask LogType="BuildCompleted" />
 
-    <Message Text="Done Building project $(MSBuildProjectFullPath) for TargetFramework=$(TargetFramework)" Importance="low" />
+    <Message Text="Done Building project $(MSBuildProjectFullPath) for TargetFramework=$(TargetFramework)" Importance="Low" />
   </Target>
   
   <Target Name="ShowMsbuildWithParameter">
-    <Message Text="Building project $(MSBuildProjectFullPath) for TargetFramework=$(TargetFramework)" Importance="low"/>
-    <Message Text="Value passed to msbuild are..." Importance="low" />
-    <Message Text="Configuration = $(Configuration)" Importance="low" />
-    <Message Text="TargetFramework = $(TargetFramework)" Importance="low" />
-    <Message Text="Platform = $(PlatformTarget)" Importance="low" />
-    <Message Text="OutputPath = $(OutputPath)" Importance="low" />
+    <Message Text="Building project $(MSBuildProjectFullPath) for TargetFramework=$(TargetFramework)" Importance="Low"/>
+    <Message Text="Value passed to msbuild are..." Importance="Low" />
+    <Message Text="Configuration = $(Configuration)" Importance="Low" />
+    <Message Text="TargetFramework = $(TargetFramework)" Importance="Low" />
+    <Message Text="Platform = $(PlatformTarget)" Importance="Low" />
+    <Message Text="OutputPath = $(OutputPath)" Importance="Low" />
   </Target>
   
   <Target Name="ShowCallOfVSTestTaskWithParameter">
-    <Message Text="Calling task Microsoft.TestPlatform.Build.Tasks.VSTestTask with following parameter..." Importance="low" />
-    <Message Text="TestFileFullPath = $(TargetPath)" Importance="low" />
-    <Message Text="VSTestSetting = $(VSTestSetting)" Importance="low" />
-    <Message Text="VSTestTestAdapterPath = $(VSTestTestAdapterPath)" Importance="low" />
-    <Message Text="VSTestFramework = $(TargetFrameworkMoniker)" Importance="low" />
-    <Message Text="VSTestPlatform = $(PlatformTarget)" Importance="low" />
-    <Message Text="VSTestTestCaseFilter = $(VSTestTestCaseFilter)" Importance="low" />
-    <Message Text="VSTestLogger = $(VSTestLogger)" Importance="low" />
+    <Message Text="Calling task Microsoft.TestPlatform.Build.Tasks.VSTestTask with following parameter..." Importance="Low" />
```

```suggestion
    <Message Text="Calling task "Microsoft.TestPlatform.Build.Tasks.VSTestTask" with following parameters..." Importance="Low" />
```

**OR**

```suggestion
    <Message Text="Calling task "VSTestTask" with following parameters..." Importance="Low" />
```

I like this one better. Namespace is always implied, It's not like we're going to change it anytime soon!